packages/seacas/libraries/ioss/src/cmake/Dependencies.cmake

```diff
@@ -1,5 +1,5 @@
 TRIBITS_PACKAGE_DEFINE_DEPENDENCIES(
   LIB_OPTIONAL_PACKAGES SEACASExodus Pamgen
-  LIB_OPTIONAL_TPLS XDMF HDF5 CGNS
+  LIB_OPTIONAL_TPLS XDMF HDF5 CGNS DataWarehouse
```

Comment:
Related to above comment, it goes here if using the `TPL_ENABLE` form.  Would go on the line above (I think) if using the `${PACKAGE_NAME}` form.